pkg/pipeline/ingest/ingest_file.go

```diff
@@ -68,7 +76,21 @@ func (ingestF *IngestFile) Ingest(process ProcessFunction) {
 			case <-ticker.C:
 				log.Debugf("ingestFile; for loop; before process")
```

Comment:
Technically we no longer call `process()`, so perhaps the comment should be updated to remove reference to `process`.